gym_electric_motor/physical_system_wrappers/__init__.py

```diff
@@ -1,4 +1,4 @@
-from .state_action_processor import StateActionProcessor
+from .physical_system_wrapper import PhysicalSystemWrapper
```

Comment:
physical_system_wrapper or _wrappers?

Reply:
wrapper is correct. The file containing the base PhysicalSystemWrapper is called physical_system_wrapper.py. The folder containing all wrappers is named physical_system_wrappers.